packages/flutter/lib/src/widgets/dismissable.dart

```diff
@@ -113,6 +114,14 @@ class Dismissable extends StatefulWidget {
   /// immediately after the the widget is dismissed.
   final Duration resizeDuration;
 
+  /// The offset threshold the item has to be dragged in order to be considered dismissed.
+  ///
+  /// Represented as a percentage. I.e. if it is 0.4, then the item has to be dragged at least
```

Comment:
Also, that's not correct use of "i.e.". It's an example. I would say "For example, ...".